fairlearn/metrics/_plots/_plot_disparities_in_metric.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, *, y_true, y_pred, sensitive_features, show_plot=True,
```

Comment:
Two points re. API here:
* I think this should be called `plot_metric_by_group`
* instead of `metric_name`, we could just allow the format where `metric` is a single item dictionary? this would be more in line with how you would get this done with `MetricFrame`.

Another idea (for another PR) is to have a method `MetricFrame.plot(show_plot=True, ax=None)` and this would be just a wrapper for that? This would be analogous to [pd.DataFrame.plot](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.html). Although there's an obvious wrinkle re. what to do if you have multiple metrics and/or control features, which we probably don't need to resolve in this PR :-)

Reply:
I like the idea for `MetricFrame.plot()`! Re. multiple metrics/control features: I personally would probably go for something like a heat map.